source/includes/stackpath/_workloads.md

```diff
@@ -222,57 +279,84 @@ curl -X POST \
-   "specs":"SP-1",
-   "persistenceStoragePath":"/lib/data/newFolder",
-   "persistenceStorageSize":1,
-   "deploymentName":"wi-root-ion",
-   "deploymentInstancePerPops":1,
-   "enableAutoScaling":false,
-   "deploymentPops": ["YYZ", "MIA", "LAX"]
+  "name":"vm-workload",
+  "slug":"vm-workload",
+  "type":"VM",
+  "image":"stackpath-edge/centos-7-cpanel:v201905241955",
+  "addAnyCastIpAddress":false,
+  "ports": [
+    {
+      "publicPort": "80",
+      "protocol": "TCP",
+      "publicPortDesc": "rule-name"
+    }
+  ],
+  "firstBootSshKey":"ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDcYr9OnzsDfYVW2I1kX/iYJ0mPG490bI5mbxbOAKPLMuWLguxRohX804j1XbwZJ+Sna+9rSfxaYA8vgd1MoYX10l9cnMLx/MMbYp4ZquauN4pGY3WoDeCqsTss3VUMW+7RFBILpU3SJTlDV02FI36D3IXb4A8XymCyU3KC99XXTfTQsuKC+WFRMsTWtklrasqCVd5yEG90i/aJc6A3TZGOYgPFNEeVYvNDaJmIkb3y4FfShoBIMgZRt0ay7SvWZUvyfvyNmK5W9ePdhZZ58R+7tQNmCzjQ4v0suWRuGJ/XL3+03w3HEsDdQx+noL+R+qAjoNFwc0spBBhJK+Q4ADqr nothing@gmail.com \nssh-rsa...",
+  "specs":"SP-1",
+  "persistenceStoragePath":"/lib/data/newFolder",
+  "persistenceStorageSize":1,
+  "deployments": [
+    {
+      "name": "deployment-name",
+      "enableAutoScaling": false,
+      "pops": ["MIA"],
+      "instancesPerPop": "1"
+    }
+  ]
 }
 ```
 
 > Request body example for a CONTAINER Workload type:
 
 ```json
 {
-   "name":"w-user-pah",
-   "type":"CONTAINER",
-   "image":"nginx:latest",
-   "addImagePullCredentialsOption":true,
-   "containerUsername":"test.username",
-   "containerPassword":"test-password",
-   "containerServer":"container.server.io",
-   "containerEmail":"mytestEmail@email.com",
-   "environmentVariableKey": "env-key",
-   "environmentVariableValue": "env-value",
-   "secretEnvironmentVariableKey":"secret-key",
-   "secretEnvironmentVariableValue":"secret-value",
-   "addAnyCastIpAddress":false,
-   "publicPort": "80",
-   "publicPortSrc": "0.0.0.0/0",
-   "publicPortDesc": "npr_root_btd",
-   "protocol": "TCP",
-   "specs":"SP-3",
-   "persistenceStoragePath":null,
-   "persistenceStorageSize":1,
-   "deploymentName":"wi-root-pxa",
-   "deploymentInstancePerPops":1,
-   "deploymentPops": ["FRA", "YYZ"],
-   "enableAutoScaling":true,
-   "cpuUtilization":50,
-   "minInstancesPerPop":1,
-   "maxInstancesPerPop":2
+	"type": "CONTAINER",
+   "name":"container-workload",
+   "slug":"container-workload",
+	 "image": "nginx",
+   "addAnyCastIpAddress": false,
+	 "ports": [
+		 {
+			 "publicPort": "80",
+			 "protocol": "TCP",
+       "publicPortDesc": "rule-name"
+		 }
+	 ],
+   "specs":"SP-1",
+   "persistenceStoragePath": "/lib/data/newFolder",
+   "persistenceStorageSize": 1,
+	 "commands": [
+		 "/bin/sh -c \"sleep 50\""
+	 ],
+	 "environmentVariables": [
+		 {
+			 "key": "USER",
+			 "value": "my-user"
+		 }
+	 ],
+	 "secretEnvironmentVariables": [
+		 {
+			 "key": "PASSWORD",
+			 "value": "my-password"
+		 }
```

Comment:
would be nice if the example followed the same formmating (tab or space)

Reply:
Reformatted [here](https://github.com/cloudops/cloudmc-api-docs/pull/307/commits/dfd9768a21c2ec3faab74f6d17ec5881e7143f5f#diff-009b71b1c0eb5e3fe379e21d571a83b999e8661251e350ecb0efb9319d7a992eR313), much have been pasted directly from Insomnia.